### PR TITLE
237 Definition of EA Stage 1

### PR DESCRIPTION
**Pull request by Nikomus**
Trello card: https://trello.com/c/U6maYkvS/237-definition-of-economic-abuse-stage-i

## Description

- Adds global styles to all layouts
- Adds some structure to definition page
- Adds Poppins fonts and a stylesheet to load them

@neontribe/sea-map

### CODE REVIEW

File: src/Page/Definition.elm [hunk 1]
```diff
@@ -27,4 +30,30 @@ update msg model =
...
 view : Model -> Html Msg
 view model =
-    div [] [ text "Definition page" ]
+    div []
+        [ header []
+            [ h1 [ css [ pageHeading ] ] [ text "[cCc] What is Economic Abuse?" ]
+            , p [ css [ pageDescription ] ] [ text "[cCc] A concise definition of economic abuse in accessible language, and that spans a few sentences so that a small intro paragraph doesn't look strange here.", text " ", a [ href "/[cCc]somewhere" ] [ text "[cCc] more information" ] ]
+            ]
+        , ul []
+            [ li [ css [ category ] ] [ h2 [] [ text "[cCc] A Category" ] ]
```
Comment (katjam) [inline src/Page/Definition.elm:39]: We can add the real categories when we abstract the copy.

File: src/Page/Definition.elm [hunk 2]
```diff
@@ -27,4 +30,30 @@ update msg model =
...
+category =
+    Css.batch
+        [ margin2 (rem 2) zero ]
+
+
+pageDescription : Style
+pageDescription =
+    Css.batch
+        [ color colours.purple
+        , fontFamilies [ "Raleway", sansSerif.value ]
```
Comment (katjam) [inline src/Page/Definition.elm:60]: Should Raleway only be for the h1 / h2 etc or is is meant for use in contextual stuff? Isn't this 'description' actually the definition? Our other pages do't have descriptions so might be more clear as 'intro' or 'definition'

Not really an issue.

File: src/Theme.elm [hunk 3]
```diff
@@ -10,3 +12,45 @@ colours =
     , teal = hex "3CC0BE"
     , grey = hex "58585A"
     }
+
+
+{-| Injects a <style> tag into the body, and can target element or
+class selectors anywhere, including outside the Elm app.
```
Comment (katjam) [inline src/Theme.elm:18]: Nice - but sounds dangerous. ;)

**Review by katjam — APPROVED** [review]
Looks great!

File: src/Page/Definition.elm (hunk 2 above)
Comment (Nikomus) [reply src/Page/Definition.elm:60]: The style guide says that Raleway should be used for page intro paragraphs, which this feels like it is. I think we may want to soften it a bit later, but we'll probably know that when there's more on the page.

**Pull request merged**